backend/substrapp/views/model.py

```diff
@@ -120,15 +119,22 @@ class ModelPermissionViewSet(PermissionMixin,
 
     queryset = Model.objects.all()
     serializer_class = ModelSerializer
-    ledger_query_call = 'queryModelDetails'
+    ledger_query_call = 'queryModelPermissions'
 
-    @action(detail=True)
-    def file(self, request, *args, **kwargs):
+    def _has_access(self, user, asset):
```

Comment:
It seems wrong to modify a private method in a child class located in a different module.

Reply:
I made it into a public method, much cleaner. Thanks!